cypress/support/config/settings.js

```diff
@@ -3196,7 +3196,9 @@ module.exports = () => ({
           },
           test: {
             paths: ['/kyrgyz/topics/cvpv9djp9qqt?renderer_env=test'],
-            enabled: true,
+            // disabling this test as it causes issues with the test for promo-topics because of the old data used
+            // conversation and decision here: https://bbc-tpg.slack.com/archives/G03T00R76KX/p1715677266122289?thread_ts=1715614758.152839&cid=G03T00R76KX
+            enabled: false,
```

Comment:
Let's just remove it, and means there's no need to include this comment / link to Slack thread

```
            paths: [],
            enabled: false,
```

Reply:
removed